### PR TITLE
docs: fix example in window.md

### PR DESCRIPTION
**Pull request by PBI-David**
I think this fixes the following although I can't test on my machine to be certain: https://github.com/vega/vega-lite/issues/8610

Also minor wording update.

### CODE REVIEW

**Review by domoritz — CHANGES_REQUESTED** [review]

File: site/docs/transform/window.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ Here we use window transform to visualize how the average MPG for vehicles have
...
+These are examples of window transforms that can be simplified with the join aggregate transform. Please refer to the [join aggregate examples](joinaggregate.html#examples).
 
 ### Percent of Total
 
 You can use the window transform to compute an aggregate and attach it to all records. In this case, you can use the [join aggregate](joinaggregate.html) instead of the window transform. Please refer to the [join aggregate examples](joinaggregate.html#examples).
 
 Here we use the window transform to derive the global sum so that we can calculate percentage.
 
-<div class="vl-example" data-name="window_percent_of_total"></div>
+<div class="vl-example" data-name="joinaggregate_percent_of_total"></div>
```
[the remark this answers is not in the document]
Comment (PBI-David) [reply site/docs/transform/window.md:127]: Ah, you're right. I think the original example was removed to be replaced with a simpler joinaggregate here: https://github.com/vega/vega-lite/pull/4554

Maybe we should just delete the section altogether now it is covered by joinaggregate.

[the remark this answers is not in the document]
Comment (PBI-David) [reply site/docs/transform/window.md]: I have updated to remove some redundancy and simplify the text.

[the remark this answers is not in the document]
Comment (domoritz) [reply site/docs/transform/window.md]: Maybe we leave things up to line 124 with the link but delete the rest.

[the remark this answers is not in the document]
Comment (domoritz) [reply site/docs/transform/window.md]: Ah, what you did looks great.

**Comment by domoritz** [comment]
Can you run prettier?

**Comment by PBI-David** [comment]
> Can you run prettier?

Unfortunately not as I don't have an environment set up. I'm editing on the web. I can't see why prettier would fail as it looks clean and is a simple line of text with not much to actually format.

File: site/docs/transform/window.md [hunk 2]
```diff
@@ -114,14 +114,6 @@ Here we use window transform to visualize how the average MPG for vehicles have
...
-These are examples of window transforms that can be simplifies with the join aggregate transform. Please refer to the [join aggregate examples](joinaggregate.html#examples).
-
 ### Percent of Total
 
-You can use the window transform to compute an aggregate and attach it to all records. In this case, you can use the [join aggregate](joinaggregate.html) instead of the window transform. Please refer to the [join aggregate examples](joinaggregate.html#examples).
-
-Here we use the window transform to derive the global sum so that we can calculate percentage.
-
-<div class="vl-example" data-name="window_percent_of_total"></div>
+The window transform *can* be used to compute an aggregate and attach it to all records in order to derive a percent of total, however, a simpler approach is to use the [join aggregate](joinaggregate.html) transform instead.
```
Comment (domoritz) [inline site/docs/transform/window.md:119]:
```suggestion
The window transform *can* be used to compute an aggregate and attach it to all records in order to derive a percent of total, however, a simpler approach is to use the [join aggregate](joinaggregate.html) transform instead.

```

**Comment by domoritz** [comment]
Thank you

**Pull request merged**